### ISSUE TITLE
Consistent handling of multiple data per grid point

### ISSUE THREAD

**Issue by bpbond** [JGCRI/RCMIP5#141]
Some models (e.g. IPSL-CM5A-MR) have multiple data at some grid points, in particular their poles. How should we handle this? Right now, the `dplyr` implementation averages these to a single point before computing spatial/temporal summaries, while the `array` implementation treats them as individuals. We need to be consistent so that we produce identical results.

See also issue #139.

**Comment by ktoddbrown** [comment]
If we address this in the loadEnsemble that it should propagate to both versions of loadCMIP5. Thoughts?

**Comment by bpbond** [comment]
Agreed, that's the obvious place. Handle it once, then never again.
